lib/bummr/outdated.rb

```diff
@@ -5,23 +5,24 @@ module Bummr
   class Outdated
     include Singleton
 
-    def outdated_gems
-      @outdated_gems ||= begin
```

Comment:
I removed the memoization here because in combination with being a singleton it made the new tests fail when run all together.